### PR TITLE
Branch fix feature flaws

### PR DESCRIPTION
**Pull request by starchypotatocode**
Please check through the comment update carefully, and let me know what is inappropriate if any

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -67,7 +67,7 @@ dependencies {
 
 
 shadowJar {
-    archiveBaseName = 'hallpointer.jar'
+    archiveBaseName = 'hallpointer'
```
Comment (keithxun) [inline build.gradle:70]: good catch

File: docs/SettingUp.md [hunk 2]
```diff
@@ -54,7 +54,7 @@ If you plan to use Intellij IDEA (highly recommended):
 
 1. **Learn the design**
 
-   When you are ready to start coding, we recommend that you get some sense of the overall design by reading about [AddressBook’s architecture](DeveloperGuide.md#architecture).
+   When you are ready to start coding, we recommend that you get some sense of the overall design by reading about [’s architecture](DeveloperGuide.md#architecture).
```
Comment (keithxun) [inline docs/SettingUp.md]: Missing replacement?

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -137,14 +137,14 @@ How the parsing works:
...
 - stores a `UserPref` object that represents the user’s preferences. This is exposed to the outside as a `ReadOnlyUserPref` objects.
 - does not depend on any of the other three components (as the `Model` represents data entities of the domain, they should make sense on their own without depending on other components)
 
 <box type="info" seamless>
 
-**Note:** An alternative (arguably, a more OOP) model is given below. It has a `Tag` list in the `AddressBook`, which `Member` references. This allows `AddressBook` to only require one `Tag` object per unique tag, instead of each `Member` needing their own `Tag` objects.<br>
+**Note:** An alternative (arguably, a more OOP) model is given below. It has a `Tag` list in the ``, which `Member` references. This allows `` to only require one `Tag` object per unique tag, instead of each `Member` needing their own `Tag` objects.<br>
```
Comment (keithxun) [inline docs/DeveloperGuide.md]: Missing

File: docs/UserGuide.md [hunk 4]
```diff
@@ -78,7 +78,7 @@ Format: `help`
 
 ### Adding a member: `add_member`
 
-Adds a member to the address book.
+Adds a member to the CCA system.
```
Comment (keithxun) [inline docs/UserGuide.md]: Use "Hall pointer" instead of "CCA system"?

File: src/main/java/hallpointer/address/logic/commands/ClearCommand.java [hunk 5]
```diff
@@ -6,13 +6,13 @@
 import hallpointer.address.model.Model;
 
 /**
- * Clears the address book.
+ * Clears the CCA system.
```
Comment (keithxun) [inline src/main/java/hallpointer/address/logic/commands/ClearCommand.java]: use "Hall pointer" instead?

File: src/main/resources/view/MainWindow.fxml [hunk 6]
```diff
@@ -2,16 +2,16 @@
...
 <?import javafx.scene.control.MenuItem?>
 <?import javafx.scene.image.Image?>
 <?import javafx.scene.layout.StackPane?>
 <?import javafx.scene.layout.VBox?>
-<?import javafx.scene.Scene?>
 <?import javafx.stage.Stage?>
-<fx:root type="javafx.stage.Stage" xmlns="http://javafx.com/javafx/17" xmlns:fx="http://javafx.com/fxml/1"
-         title="Hall Pointer" minWidth="450" minHeight="600" onCloseRequest="#handleExit">
+
+<fx:root minHeight="600" minWidth="450" onCloseRequest="#handleExit" title="Hall Pointer" type="javafx.stage.Stage" xmlns="http://javafx.com/javafx/18" xmlns:fx="http://javafx.com/fxml/1">
```
Comment (keithxun) [inline src/main/resources/view/MainWindow.fxml]: What does changes in this file do? May be good to add a screenshot of the UI?

File: docs/SettingUp.md (hunk 2 above)
Comment (starchypotatocode) [reply docs/SettingUp.md]: Good catch

File: docs/DeveloperGuide.md (hunk 3 above)
Comment (starchypotatocode) [reply docs/DeveloperGuide.md]: Fixed

File: docs/UserGuide.md (hunk 4 above)
Comment (starchypotatocode) [reply docs/UserGuide.md]: Consistency though? (Elsewhere uses CCA system iirc)

File: src/main/java/hallpointer/address/model/session/Session.java [hunk 7]
```diff
@@ -8,7 +8,7 @@
 import hallpointer.address.model.point.Point;
 
 /**
- * Represents a Session in the system.
+ * Represents a Sessionsystem.
```
Comment (keithxun) [inline src/main/java/hallpointer/address/model/session/Session.java:11]: This sounds weird

**Pull request merged**